samples/snippets/feature_store_service/batch_create_features_sample_test.py

```diff
@@ -15,8 +15,8 @@
 import os
 from uuid import uuid4
 
-import batch_create_features_sample
-import create_entity_type_sample
+import batch_create_features_sample_v1
+import create_entity_type_sample_v1
```

Comment:
```
import batch_create_features_sample
import create_entity_type_sample
```

Reply:
Thanks for the comments. Instead of directly upgrading the existing snippets, we actually need to keep the current v1beta1 snippets and add another v1 version. This is to ensure that our customers using the v1beta1 API can still reference the old API docs.